workflow/cases/coupled_free_forecast_s2s.yaml

```diff
@@ -0,0 +1,53 @@
+case:
+  places:
+    workflow_file: layout/free_forecast_gfs.yaml
+    MOM6IC: /scratch1/NCEPDEV/nems/Bin.Li/S2S/FROM_HPSS
```

Comment:
In PR 80 this variable was renamed and the contents of the file was moved to: 
BASE_CPLIC: /scratch2/NCEPDEV/climate/climpara/S2S/IC

Reply:
Thanks. I updated it. @JessicaMeixner-NOAA